apps/pipelines/models.py

```diff
@@ -67,6 +68,37 @@ def version_display(self) -> str:
             return ""
         return f"v{self.version_number}"
 
+    @classmethod
+    def create_default(cls, team):
+        from apps.pipelines.nodes.nodes import EndNode, StartNode
+
+        default_name = "New Pipeline"
+        existing_pipeline_count = cls.objects.filter(team=team, name__startswith=default_name).count()
+
+        start_node = {
+            "id": str(uuid4()),
+            "type": "pipelineNode",
```

Comment:
What about changing this type to something else and having a completely different rect component for the start & end nodes? We could do away with the 'ui:show_input" and "ui:show_output" schema customizations and we would have easier control over the style of those nodes e.g. we could make them round, not show the toolbar etc.

Reply:
I updated this to use `startNode` and `endNode`, with different styling for each. 

![image](https://github.com/user-attachments/assets/5e9b1a3f-e3ad-4685-80b8-01663f0c40f7)